sdk/eventgrid/Azure.Messaging.EventGrid/src/Azure.Messaging.EventGrid.csproj

```diff
@@ -7,7 +7,7 @@
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <EnableApiCompat>false</EnableApiCompat>
     <!-- These supressions should be removed in a production library -->
-    <NoWarn>$(NoWarn);CA1812</NoWarn>
+    <NoWarn>$(NoWarn);CA1812, CS1591, CA1801</NoWarn>
```

Comment:
Were these brought based on Anne's tutorial? 
As of PR https://github.com/Azure/azure-sdk-for-net/pull/12977, we no longer have any of these NoWarn in the template project, so we may want to match that.